sdk/core/src/main/java/com/klaviyo/coresdk/model/UserInfo.kt

```diff
@@ -10,7 +10,7 @@ import java.util.UUID
 /**
  * Stores information on the currently active user
  */
-internal object UserInfo {
+object UserInfo {
```

Comment:
Remind me (cause I am sure you told me before), but why are we exposing `UserInfo` now?

Reply:
No worries, I don't think I did. I realized I needed access to `UserInfo.getAsProfile` from the `push` module in order to be able to call `enqueuePushToken(token: String, profile: Profile)` with the correct identifiers. Previously, I was just using `Klaviyo.setProfile`, so didn't need to access `UserInfo` directly. 

Alternatively I could add a method `Klaviyo.setPushToken` but I worried about having that be in the public API of the _analytics_ module and getting mixed up with the push module.

Ultimately, `UserInfo` will be private when we split the `core` and `analytics` into two modules though.

Reply:
Actually the more I think about it, it'd make sense to put `setPushToken` in the `analytics` module and call to it from `push-fcm`, as we'd want to use the same method from, say a `push-fireos` module in the future. Indeed if the app developer is implementing their firebase message handler entirely in their own code, they wouldn't need our `push-fcm` package at all.

Reply:
If you don't mind, I'm going to put a TODO on this and otherwise handle this question in the gradle organizing branch.

Reply:
All good points, ya I think a TODO is the best call here! Just make sure we properly note down these thoughts into the SDK or a ticket and we'll be able to revisit this easily in the future. It's an abstraction detail so I don't foresee any issues with making changes post-release.

Reply:
Cool, added TODO and a comment to the gradle TP ticket about